requests/adapters.py

```diff
@@ -419,8 +419,9 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
 
             # Send the request.
             else:
-                if hasattr(conn, 'proxy_pool'):
-                    conn = conn.proxy_pool
+                proxy_pool = getattr(conn, 'proxy_pool', None)
+                if proxy_pool is not None:
+                    conn = proxy_pool
```

Comment:
Ok, so now I'm looking closer.

We can translate this to `getattr(conn, 'proxy_pool', conn)`.